### ISSUE TITLE
[2.6] Not receiving BGs (from a Patched Dexcom, at least)

### ISSUE THREAD

**Issue by lee-b** [MilosKozak/AndroidAPS#2503]
[AndroidAPS.log](https://github.com/MilosKozak/AndroidAPS/files/4328837/AndroidAPS.log)
Since installing 2.6, AAPS doesn't receive BG data from a Patched Dexcom app.

The previous dexcom app version, that worked with the previous AAPS versions, didn't work with this version.  I then tried updating to Dexcom app v1.5.3.4 ("custom dexcom" version; international, mmol/mol).  Same problem: the Dexcom app has data, but AAPS isn't receiving it.

I have XDrip sharing (from AAPS) configured too, and XDrip configured for Eversense  / 640g.

Neither of these apps are seeing BGs, though the Dexcom app seems fine.

I've tried switching to XDrip as a BG source, with XDrip getting its data from Dexcom Follower, but this seems to need a login / password, that the app no longer uses since it auths via its own email invite method.

I'm using the patched Dexcom App since direct G6 support in XDrip doesn't calibrate new sensors  (open issue, making no real progress), and XDrip, although it connects occasionally, lacks connection for much longer time periods than the Dexcom app.

In short, since 2.6, AAPS looping is completely broken for me, with a Dexcom G6.

Build version: c3dbd1ec2-2020.03.08-15:02
Flavor: fullrelease
Nightscout version: 0.11.1

Log attached.

**Comment by twain47** [comment]
Could you confirm where you are getting your patched dexcom app from?

It should be here: https://github.com/dexcomapp/dexcomapp/tree/master/2.4

**Comment by lee-b** [comment]
I believe it was from here:

https://docs.google.com/forms/d/e/1FAIpQLScD76G0Y-BlL4tZljaFkjlwuqhT83QlFM5v6ZEfO7gCU98iJQ/viewform?fbzx=-669169667410876002

I can attach the actual APK (or put it somewhere and link it) if that helps.

**Comment by guydavies** [comment]
Hi Lee,

That one doesn’t support broadcasting to other apps.  You must use the one to which Brian provided the link.

Rgds,

Guy

**Comment by lee-b** [comment]
Ah, OK.  Apologies.

Working now :)

Thanks @guydavies, and @twain47
